src/NServiceBus.AcceptanceTesting/ScenarioContext.cs

```diff
@@ -28,10 +29,17 @@ public void AddTrace(string trace)
 
         ConcurrentQueue<string> traceQueue = new ConcurrentQueue<string>();
 
+        internal Dictionary<string, LogLevel> LogLevels = new Dictionary<string, LogLevel>();
+
+        public void SetLogLevel(string logger, LogLevel level)
```

Comment:
does the log level need to be scoped to the specific logger? Wouldn't it be easier to just configure a log level for all loggers?